docs/iceberg-rest-service.md

```diff
@@ -159,3 +159,6 @@ DESCRIBE TABLE EXTENDED dml.test;
 INSERT INTO dml.test VALUES (1), (2);
 SELECT * FROM dml.test
 ```
+
+For the details, review the
```

Comment:
change it to this?
`You could try Spark with Gravitino REST catalog service in our [playground](./how-to-use-the-playground.md#using-iceberg-rest-service)`

Reply:
@qqqttt123 please fix this.

Reply:
Fixed.